nsv13/code/modules/overmap/camera.dm

```diff
@@ -65,6 +65,9 @@
 		M.client.check_view()
 	var/mob/camera/aiEye/remote/overmap_observer/eyeobj = M.remote_control
 	M.cancel_camera()
+	if(M.client) //Reset px, y
```

Comment:
why is this changed just asking

Reply:
It resolves a weird issue where your screen would be skewed / offset when stopping piloting ships.